zerver/models/realms.py

```diff
@@ -898,7 +898,7 @@ def display_subdomain(self) -> str:
         return self.string_id
 
     @property
-    def uri(self) -> str:
+    def url(self) -> str:
         return settings.EXTERNAL_URI_SCHEME + self.host
```

Comment:
I found a couple missing hunks:
```
$ git grep realm[.]uri
templates/corporate/support/realm_details.html:    <b>URL</b>: <a target="_blank" rel="noopener noreferrer" href="{{ realm.uri }}">{{ realm.uri }}</a> |
zerver/migrations/0501_delete_dangling_usermessages.py:            # Reimplement realm.uri
```

Always take the time to use `git grep` to check your work when rebasing an old PR like this.

Reply:
I actually tried different patterns with `git grep` but I don't know how I missed such an obvious search. I have fixed it now.